src/dialects/oracle/index.js

```diff
@@ -54,7 +54,7 @@ OracleDialect.prototype.supports = _.merge(_.cloneDeep(AbstractDialect.prototype
   GEOMETRY: false
 });
 
-OracleDialect.prototype.defaultVersion = '18.4.0';
+OracleDialect.prototype.defaultVersion = '18.0.0';
```

Comment:
With this, maybe we should test on the first version of 18 then? So the oldest minor version we support and the latest minor version we support.

Reply:
The oldest minor version for Oracle DB 18 is 18.0.0 and we are testing against this version of the DB.  
The latest major version for Oracle DB is 21.3.0 and we're also testing against it.

Reply:
Thanks!